server/views/pages/induction/prePrisonEducation/qualificationLevel.njk

```diff
@@ -19,111 +19,119 @@ Data supplied to this template:
+            },
+            {
+              value: "LEVEL_5",
+              text: "LEVEL_5" | formatQualificationLevel,
+              hint: {
+                text: "LEVEL_5" | formatQualificationLevelHint
               },
-              {
-                value: "LEVEL_5",
-                text: "LEVEL_5" | formatQualificationLevel,
-                hint: {
-                  text: "LEVEL_5" | formatQualificationLevelHint
-                },
-                checked: form.qualificationLevel === "LEVEL_5"
+              checked: form.qualificationLevel === "LEVEL_5"
+            },
+            {
+              value: "LEVEL_6",
+              text: "LEVEL_6" | formatQualificationLevel,
+              hint: {
+                text: "LEVEL_6" | formatQualificationLevelHint
               },
-              {
-                value: "LEVEL_6",
-                text: "LEVEL_6" | formatQualificationLevel,
-                hint: {
-                  text: "LEVEL_6" | formatQualificationLevelHint
-                },
-                checked: form.qualificationLevel === "LEVEL_6"
+              checked: form.qualificationLevel === "LEVEL_6"
+            },
+            {
+              value: "LEVEL_7",
+              text: "LEVEL_7" | formatQualificationLevel,
+              hint: {
+                text: "LEVEL_7" | formatQualificationLevelHint
               },
-              {
-                value: "LEVEL_7",
-                text: "LEVEL_7" | formatQualificationLevel,
-                hint: {
-                  text: "LEVEL_7" | formatQualificationLevelHint
-                },
-                checked: form.qualificationLevel === "LEVEL_7"
+              checked: form.qualificationLevel === "LEVEL_7"
+            },
+            {
+              value: "LEVEL_8",
+              text: "LEVEL_8" | formatQualificationLevel,
+              hint: {
+                text: "LEVEL_8" | formatQualificationLevelHint
               },
-              {
-                value: "LEVEL_8",
-                text: "LEVEL_8" | formatQualificationLevel,
-                hint: {
-                  text: "LEVEL_8" | formatQualificationLevelHint
-                },
-                checked: form.qualificationLevel === "LEVEL_8"
-              }
-            ],
-            errorMessage: errors | findError('qualificationLevel')
-          }) }}
-        </div>
+              checked: form.qualificationLevel === "LEVEL_8"
+            }
+          ],
+          errorMessage: errors | findError('qualificationLevel')
+        }) }}
 
         {{ govukButton({
-            id: "submit-button",
-            text: "Continue",
-            type: "submit",
-            attributes: {"data-qa": "submit-button"}
-          }) }}
-      </form>
+          id: "submit-button",
+          text: "Continue",
+          type: "submit",
+          attributes: {"data-qa": "submit-button"}
+        }) }}
+      </div>
+
+      <div class="govuk-grid-column-one-third">
```

Comment:
This '1 third' div that contains the related content sidebar is a sibling to the '2 thirds' div on line 37 , which is how the sidebar is lining up horizontally with the top of the radio set 👍